pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1268,6 +1297,29 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 				if err != nil {
 					return err
 				}
+			} else if iface.Vhostuser != nil {
+				domainIface.Type = "vhostuser"
+				interfaceName := GetPodInterfaceName(networks, cniNetworks, iface.Name)
+				vhostPath, vhostMode, err := getVhostuserInfo(interfaceName, c)
```

Comment:
I think it is better to set MAC address here if CNI returns a mac.

Reply:
userspace cni does not provide mac address. It is not possible to validate it without userspace cni. In this iteration, the mac address is left with libvirt auto generated. In the next iteration, the support will be added in both cni and kubevrit to support custom mac address.